packages/core/src/components/divider/divider.tsx

```diff
@@ -9,6 +9,6 @@ export class AtomDivider {
   @Prop() type: 'horizontal' | 'vertical' = 'horizontal'
 
   render() {
-    return <div data-type={this.type} class='atom-divider' />
+    return <div class={`atom-divider ${this.type}`} aria-hidden={true} />
```

Comment:
suggestion: maybe we could use `is-` syntax, like `is-vertical`

```suggestion
    return <div class={`atom-divider is-${this.type}`} aria-hidden={true} />
```